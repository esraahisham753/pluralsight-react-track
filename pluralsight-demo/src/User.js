import React from 'react';

export default function User(props) {
    return (
        <div>
            <h1>User details</h1>
            Name: {props.name}
            Age: {props.age}
        </div>
    );
}