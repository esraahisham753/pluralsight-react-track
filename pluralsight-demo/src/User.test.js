import React from 'react';
import renderer from 'react-test-renderer';
import User from './User';


it('renders correctly', () => {
    const tree = renderer.create(<User name="Cory" age="old enough" />).toJSON();

    expect(tree).toMatchSnapshot();
});