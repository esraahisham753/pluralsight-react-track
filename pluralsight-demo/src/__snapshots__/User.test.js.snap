// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    User details
  </h1>
  Name: 
  Cory
  Age: 
  old enough
</div>
`;
